packages/circuit-ui/components/TopNavigation/components/UtilityLinks/UtilityLinks.spec.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CircleMore } from '@sumup/icons';
+
+import { axe, render, renderToHtml } from '../../../../util/test-utils';
+
+import { UtilityLinks } from './UtilityLinks';
+
+describe('UtilityLinks', () => {
+  const baseProps = {
+    links: [
+      {
+        icon: CircleMore,
+        label: 'More',
+        href: '/more',
+        onClick: jest.fn(),
+      },
+    ],
+  };
+
+  describe('styles', () => {
+    it('should match the snapshot', () => {
```

Comment:
Wonder if we should add more logic tests generally (for example in this case clicking the thing should call the onClick). What do you think? Any reason you went for snapshots only?

Reply:
Sure, will do.

Reply:
Done in cf51daa.